nwb_conversion_tools/datainterfaces/giocomoconverter.py

```diff
@@ -0,0 +1,43 @@
+from ..nwbconverter import NWBConverter
+from .imagingextractorinterface import SbxImagingInterface
+from .roiextractordatainterface import Suite2pSegmentationInterface
+from .giocomovrdatainterface import GiocomoVRInterface
```

Comment:
The Giocomo converter should go in [giocomo-lab-to-nwb](https://github.com/catalystneuro/giocomo-lab-to-nwb)